components/editor/TimesheetEntryEditor.tsx

```diff
@@ -149,6 +163,7 @@ const DayEditor = ({
                             deleteTimesheetEntry={del}
                             dateStr={dateStr}
                             timeCategories={timeCategories}
+                            tasks={tasks}
```

Comment:
A small error: This should be `taskByProject` wrt. the project id corresponding to `entry`. I.e. if `entry` is in project `A` this should not contain tasks from project `B`.